vertical-pod-autoscaler/hack/vpa-process-yamls.sh

```diff
@@ -38,7 +38,7 @@ if [ $# -gt 2 ]; then
   exit 1
 fi
 
-YAMLS="api/vpa-crd.yaml deploy/vpa-rbac.yaml deploy/admission-controller-rbac.yaml deploy/updater-deployment.yaml deploy/recommender-deployment.yaml deploy/admission-controller-deployment.yaml"
```

Comment:
I think you meant to remove the admission-contraoller-rbac.yaml?

Reply:
Good catch. Thanks!